tests/jointcalTestBase.py

```diff
@@ -255,3 +258,75 @@ def _test_metrics(self, result, expect):
                     self.assertFloatsAlmostEqual(value, expect[key.metric], msg=key.metric, rtol=1e-5)
                 else:
                     self.assertEqual(value, expect[key.metric], msg=key.metric)
+
+    def _importRepository(self, instrument, exportPath, exportFile):
+        """Import a gen3 test repository into self.testDir
+
+        NOTE: this is lifted from Eli's work on DM-24318, and should be moved
+        to a common place, maybe pipe_base?
```

Comment:
I'm not sure how common these calls are, and it's 5 lines, I'm not sure if it's worth promoting to pipe_base yet with only two usages?